plugins/modules/purefa_info.py

```diff
@@ -166,9 +167,17 @@ def generate_default_dict(module, array):
                 default_info["encryption_algorithm"] = encryption.data_at_rest.algorithm
                 default_info["encryption_module_version"] = encryption.module_version
             eradication = array_data.eradication_config
-            default_info["eradication_days_timer"] = int(
-                eradication.eradication_delay / SEC_TO_DAY
-            )
+            if OBJECT_SAFE_MODE_API_VERSION in api_version:
+                default_info["eradication_safe_mode_disabled_days_timer"] = int(
+                    eradication.disabled_delay / SEC_TO_DAY
+                )
+                default_info["eradication_safe_mode_enabled_days_timer"] = int(
+                    eradication.enabled_delay / SEC_TO_DAY
+                )
+            else:
+                default_info["eradication_days_timer"] = int(
```

Comment:
`eradication_delay` is still a valid option for REST 2.26, depending on how the array is configured, so I would not have this as an `else` and use `getattr` to provide a value of `None` if it isn't provided, then do the `SEC_TO_DAY` conversion if it does exist.

Reply:
That is a good point. I will get this updated shortly.